package.json

```diff
@@ -14,37 +14,41 @@
     ".prettierrc.js"
   ],
   "scripts": {
+    "prepare": "husky install",
+    "test": "npm run lint",
     "format": "prettier . --write",
     "lint": "npm-run-all --parallel lint:*",
     "lint:formatting": "prettier . --check --ignore-path .gitignore",
     "lint:js": "eslint . --ignore-path .gitignore",
     "release": "np"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
   "lint-staged": {
     "*.js": "eslint --cache --fix",
     "*.{js,json,md,yml}": "prettier --write"
   },
   "eslintConfig": {
     "extends": [
       "stylelint"
-    ]
+    ],
+    "rules": {
+      "jest/no-deprecated-functions": "off"
```

Comment:
[note] This turning off is to prevent the error below of ESLint:

```
Error: Error while loading rule 'jest/no-deprecated-functions': Unable to detect Jest version - please ensure jest package is installed, or otherwise set version explicitly
```